app/lib/submission_builder/ty2022/states/az/individual_return.rb

```diff
@@ -39,10 +39,6 @@ def self.state_abbreviation
             "AZ"
```

Comment:
Out of curiosity, is this being refactored out in a separate PR? It seems like it would follow a similar pattern to this change.

Reply:
Nevermind, I see that this is the next PR you sent!